### PR TITLE
FLEX-3420 ~ Tests new light measurement device type.

### PR DESCRIPTION
**Pull request by kevinsmeets**
[@Common @Platform @CoreDeviceManagement]

### CODE REVIEW

**Comment by kevinsmeets** [comment]
This pull request can run it's tests after the new version of OSGP  has been deployed to osgp-automatic-tst.aws.osg-platform.com

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/database/core/SsldDeviceSteps.java [hunk 1]
```diff
@@ -67,6 +67,12 @@ public Ssld aDevice(final Map<String, String> settings) throws Throwable {
         return this.createAnSsldDevice(settings);
     }
 
+    @Given("^the light measurement devices$")
+    @Transactional("txMgrCore")
+    public void theLightMeasurmentDevices() throws Throwable {
```
Comment (rlemmers) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/database/core/SsldDeviceSteps.java]: theLightMeasurmentDevices => theLightMeasur**e**mentDevices

File: cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-core/DeviceManagement/FindDevices.feature [hunk 2]
```diff
@@ -14,3 +14,22 @@ Feature: CoreDeviceManagement Find Devices
...
+    When receiving a find devices request
+      | DeviceIdentification       | Lichtmeter*          |
+      | HasTechnicalInstallation   | true                 |
+      | PageSize                   |                    5 |
+      | Page                       |                    0 |
+      | SortDir                    | asc                  |
+      | SortedBy                   | deviceIdentification |
+    Then the find devices response contains "4" devices
+    And the find devices response contains at index "1"
+      | DeviceIdentification | LichtmeterLeeuwarden |
```
Comment (rlemmers) [inline cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-core/DeviceManagement/FindDevices.feature]: Replace the device identifications for the light measurement devices by dummy names. Also adapt this in the BaseDeviceSteps.

**Comment by kevinsmeets** [comment]
retest this please

**Pull request merged**